src/widget.rs

```diff
@@ -7,6 +7,15 @@ use serde_json::value::Value;
 
 use crate::themes::Theme;
 
+#[derive(Debug, Copy, Clone, Deserialize)]
+pub enum Spacing {
+    Normal,
```

Comment:
Just to make it clearer: "Adds a leading and trailing space around the widget contents"